<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TableLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Coffeemachine usage log</string>
    </property>
   </widget>
   <widget class="QListWidget" name="coffeelog">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>441</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxRecord">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>431</width>
      <height>151</height>
     </rect>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="nameEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Fname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>First name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnameEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="LName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Second name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="patronEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Patron">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Patronymic</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="degreeEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Degree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Degree</string>
     </property>
    </widget>
    <widget class="QLabel" name="difference">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Coffee difference</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="diffEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>49</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="DateAndTime">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="positionEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Position">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="NewRecord">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create a record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeRecord">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change current record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveChanges">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save changes</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="CheckRecordGroup">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="diffEditCheck">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="differenceCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Coffee remains</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="DateAndTimeCheck">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBoxRecord">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCheckRecord">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Check record</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="CreateGroup">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="IndexLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Index</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxIndex">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="CreateButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete current record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveInFileButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadFromFileButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load from</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="saveEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="loadEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="verifyButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>
   <widget class="QLabel" name="verification">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Coffeemachine</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Remain</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="remEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
